spec/system/support_interface/managing_users_v2/adding_multiple_provider_users_spec.rb

```diff
@@ -0,0 +1,189 @@
+require 'rails_helper'
+
+RSpec.feature 'Managing provider users v2' do
```

Comment:
maybe we can have two system specs one for a single user and one for multiple users?

Reply:
Yep sure 👍

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4896/commits/4e17698d5a32d28d5a5c6200397838b05eb68705